### PR TITLE
fix(rowactioncell): change action color

### LINKED ISSUE #1314: [RowActionCell] Colors don't adhere to design
### Describe the bug

Colors were incorrect according the component design

See internal issue: https://github.ibm.com/wiotp/monitoring-dashboard/issues/1043

### Environment/versions:

- OS: MacOS
- Browser: chrome
- `carbon-addons-iot-react` version: v2.90

### PR DESCRIPTION
**Pull request by sstone2423**
Closes #1314 

**Summary**

- Color was blue by default which did not adhere to the original design

**Change List (commits, features, bugs, etc)**

- Swap color to black by default and blue on hover

**Acceptance Test (how to verify the PR)**

- Check the row actions story to see that the colors are correct

Screen Shot 2020-06-16 at 12 48 37 PM

### CODE REVIEW

**Comment by cynthiaz** [comment]
@sstone2423 
I may have been misguided as well regarding the color for the hover state. I checked again Carbon's guideline. To clarify, here's the 3 kinds of row action buttons and the color for each state

image

To correct myself, the icon button should be black in both default and hover state, see carbon's example: https://react.carbondesignsystem.com/?path=/story/datatable--with-overflow-menu

**Comment by CynZhang** [comment]
@sstone2423 
I may have been misguided as well regarding the color for the hover state. I checked again Carbon's guideline. To clarify, here's the 3 kinds of row action buttons and the color for each state

image

To correct myself, the icon button should be black in both default and hover state, see carbon's example: https://react.carbondesignsystem.com/?path=/story/datatable--with-overflow-menu

File: src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss [hunk 1]
```diff
@@ -11,6 +11,7 @@
   }
   .#{$iot-prefix}--row-actions-cell-btn--icononly {
     margin-left: 0;
+    color: black; // black by default
```
Comment (tay1orjones) [inline src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss:14]: Looks like it's Gray 100, not straight black

```suggestion
    color: $ui-05;
```

image

Comment (tay1orjones) [reply src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss:14]: Actually with the `color` property it should probably be `$text-01`

**Comment by tay1orjones** [comment]
@sstone2423 @CynZhang could you point me to a story in our storybook that shows this defect? This story shows a black overflow menu item and the other action buttons aren't iconOnly, so they're blue as intended 
https://carbon-addons-iot-react.com/?path=/story/watson-iot-table--with-row-expansion-and-actions

**Comment by CynZhang** [comment]
@tay1orjones I believe the one you posted there is the one, and the requested change was to update the icon's hover state color to $text-01 instead of blue. 
The other one I found is https://carbon-addons-iot-react.com/?path=/story/watson-iot-tablebodyrow--row-actions-with-overflow  which also uses Blue color for icon buttons.

**Comment by tay1orjones** [comment]
Thanks @CynZhang - that's a great example I overlooked.

@sstone2423 I believe we should be using `hasIconOnly` for these buttons so we provide the optimal accessibility affordance via the tooltip in addition to the color consistency.

**Comment by sstone2423** [comment]
Good call. Didn't realize there was an iconOnly button already! That makes it way simpler. Looks like its naturally taking the gray100 now

File: src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss (hunk 1 above)
Comment (sstone2423) [reply src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss]: This style is no longer needed. Using `hasIconOnly` naturally gave the icon the correct color again

**Pull request merged**